### PR TITLE
Add r-cmprskcoxmsm and r-twang

### PR DESCRIPTION
**Pull request by leej3**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-cmprskcoxmsm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  home: https://CRAN.R-project.org/package=cmprskcoxmsm
+  license: GPL-2.0-or-later
+  summary: 'Uses inverse probability weighting methods to estimate treatment effect under marginal
+    structure model for the cause-specific hazard of competing risk events. Estimates
+    also the cumulative incidence function (i.e. risk) of the potential outcomes, and
+    provides inference on risk difference and risk ratio. Reference: Kalbfleisch & Prentice
+    (2002)<doi:10.1002/9781118032985>; Hernan et al (2001)<doi:10.1198/016214501753168154>.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-cmprskcoxmsm/meta.yaml:55]: Expand license consistent with CRAN:

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

File: recipes/r-twang/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    - r-base
+    - r-matrixmodels
+    - r-data.table
+    - r-gbm >=1.5_3
+    - r-ggplot2
+    - r-lattice
+    - r-latticeextra
+    - r-survey
+    - r-xgboost
+    - r-xtable
```
Comment (mfansler) [inline recipes/r-twang/meta.yaml:44]: Makevars shows they link against BLAS/LAPACK. Even though it didn't show an issue, it won't hurt to have this in the metadata.

```suggestion
    - r-xtable
    - libblas
    - liblapack
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for adding these recipes! I've suggested a couple minor changes, but then should be good.

**Comment by leej3** [comment]
Thanks for the review and the pointers.

**Review by mfansler — APPROVED** [review]

**Pull request merged**